.prettierignore

```diff
@@ -0,0 +1,6 @@
+**/.next/**
+**/_next/**
+.github/
+bazel-*
+node_modules
+pnpm-lock.yaml
```

Comment:
This should also include the `out` directory, which is generated on `npm run build` (or `export`?). Without that `npm run prettier-check` fails.

Reply:
Ah yeah, thx, done